engines/query-sparql/config/config-default.json

```diff
@@ -3,6 +3,6 @@
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^4.0.0/components/context.jsonld"
   ],
   "import": [
-    "ccqs:config/config-default-v4-1-0.json"
+    "ccqs:config/config-default-v4-2-0.json"
```

Comment:
Similar to this file, the other 3 engines will also require a reference to their `-v4-2-0` variant.